--- civi-download-tools.orig	2019-08-02 09:45:49.386629587 -0400
+++ civi-download-tools	2019-08-02 09:45:33.531116011 -0400
@@ -337,7 +337,7 @@
     source /etc/lsb-release
   fi
   case "$DISTRIB_CODENAME" in
-    precise|trusty|jessie|stretch|xenial|yakkety|zesty|artful|bionic|cosmic|disco)
+    precise|trusty|buster|stretch|xenial|yakkety|zesty|artful|bionic|cosmic|disco)
       echo "do_system_$DISTRIB_CODENAME"
       ;;
     *)
@@ -515,18 +515,16 @@
 }
 
 ###############################################################################
-function do_system_jessie() {
+function do_system_buster() {
   set -e
-    PACKAGES="acl git wget unzip zip mysql-server mysql-client php5-cli php5-imap php5-ldap php5-curl php5-mysql php5-intl php5-gd php5-mcrypt php-apc apache2 libapache2-mod-php5 nodejs-legacy npm"
+    PACKAGES="acl git wget unzip zip default-mysql-server default-mysql-client php-cli php-imap php-ldap php-curl php-mysql php-intl php-gd php-apcu apache2 libapache2-mod-php php-bcmath php-mbstring php-soap php-zip php-xml php-curl nodejs npm"
     echo "Detected \"Debian Jessie\"."
     echo ""
     echo "Recommended packages: $PACKAGES"
     echo ""
     if cvutil_confirm "Run automated installation? [Y/n] " y y; then
-      wget -O- https://deb.nodesource.com/setup_8.x | sudo -E bash -
       sudo apt-get -y install $PACKAGES
-      sudo php5enmod mcrypt
-      sudo php5enmod imap
+      sudo phpenmod imap
       sudo a2enmod rewrite
       sudo apache2ctl restart
     else
